core/templates/dev/head/css/oppia.css

```diff
@@ -1189,10 +1189,27 @@ pre.oppia-pre-wrapped-text {
   position: relative;
 }
 
-.oppia-editable-section:hover .oppia-state-content-display,
-.oppia-editable-section:hover .oppia-rule-preview {
```

Comment:
I think it will be good idea if I dont touch the 
.oppia-rule-preview class, I will add it back again in next commit.